sdk/nodejs/asset/archive.ts

```diff
@@ -23,14 +23,14 @@ export abstract class Archive {
      * A private field to help with RTTI that works in SxS scenarios.
      */
     // tslint:disable-next-line:variable-name
-    /* @internal */ private readonly __pulumiArchive: boolean = true;
+    /* @internal */ public readonly __pulumiArchive: boolean = true;
```

Comment:
explained below.  importantly, this is not a visible change to users outside of the `@pulumi/pulumi` package.